post-processing/post_processing.py

```diff
@@ -23,12 +23,13 @@ def __init__(self, log_path: Path, debug=False, verbose=False):
         # dataframe filters
         self.mask = pd.Series(self.df.index.notnull())
 
-    def run_post_processing(self, config: ConfigHandler):
+    def run_post_processing(self, log_path: Path, config: ConfigHandler):

```

Comment:
I have one more note about this - it would be preferable to no longer need a `log_path` as an input to the `run_post_processing` function since refactoring specifically moved it to only be used by `init`. Since this is now needed here again, perhaps we can just store it in the class during `init` and use `self.log_path` in `run_post_processing`?

Reply:
Actually, it's not a must to export the `.csv` file under the `perflogs` folder but I thought it would be good to have everything related with perflogs in the same folder. 

The easiest solution is to export it to the directory we run the benchmark I guess without any modification to functions?

Reply:
Yes, that also works, but as you said it might be better to keep perflog-related things together in one folder. It's completely fine to use `self.log_path` in `run_post_processing` (just throw a `self.log_path = log_path` into `init` since we need a `log_path` to initialise the `PostProcessing` class anyway and we know the path has to exist), I only wanted to remove it as an argument because it makes stitching together the front-end more convenient.

Reply:
I think we should keep all the output in one place, and not mix it with the perflogs. This place is currently the directory where we run the benchmark. If we want to control the output location, we can add an `output_path` to the postprocessing arguments - but please open a separate issue for this!

Reply:
Opened issue #306